fieldservice/security/ir.model.access.csv

```diff
@@ -1,32 +1,53 @@
 access_fsm_region_user,region.user,model_fsm_region,fieldservice.group_fsm_user,1,0,0,0
 access_fsm_region_manager,region.manager,model_fsm_region,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_region_portal,access.fsm.region.portal,model_fsm_region,base.group_portal,1,0,0,0
 access_fsm_equipment_fsm_user,fsm.equipment.user,model_fsm_equipment,fieldservice.group_fsm_user,1,0,0,0
 access_fsm_equipment_fsm_manager,fsm.equipment.manager,model_fsm_equipment,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_equipment_portal,access.fsm.equipment.portal,model_fsm_equipment,base.group_portal,1,0,0,0
 access_fsm_category_user,fsm.category.user,model_fsm_category,fieldservice.group_fsm_user,1,0,0,0
 access_fsm_category_manager,fsm.category.manager,model_fsm_category,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_category_portal,access.fsm.category.portal,model_fsm_category,base.group_portal,1,0,0,0
 access_fsm_template_user,fsm.template.user,model_fsm_template,fieldservice.group_fsm_user,1,0,0,0
 access_fsm_template_manager,fsm.template.manager,model_fsm_template,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_template_portal,access.fsm.template.portal,model_fsm_template,base.group_portal,1,0,0,0
 access_fsm_team_user,fsm.team.user,model_fsm_team,fieldservice.group_fsm_user,1,0,0,0
 access_fsm_team_manager,fsm.team.manager,model_fsm_team,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_team_portal,access.fsm.team.portal,model_fsm_team,base.group_portal,1,0,0,0
 access_fsm_location_person_user,fsm.location.person.user,model_fsm_location_person,fieldservice.group_fsm_user,1,1,0,0
 access_fsm_location_person_manager,fsm.location.person.manager,model_fsm_location_person,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_location_person_portal,access.fsm.location.person.portal,model_fsm_location_person,base.group_portal,1,0,0,0
 access_fsm_order_type_user,fsm.order.type.user,model_fsm_order_type,fieldservice.group_fsm_user,1,1,0,0
 access_fsm_order_type_manager,fsm.order.type.manager,model_fsm_order_type,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_order_type_portal,access.fsm.order.type.portal,model_fsm_order_type,base.group_portal,1,0,0,0
 access_fsm_calendar_filter,fsm.calendar.filter.user,model_fsm_person_calendar_filter,fieldservice.group_fsm_user,1,1,1,1
+access_resource_resource_portal,access.resource.resource.portal,resource.model_resource_resource,base.group_portal,1,0,0,0
+access_ir_sequence_portal,access.ir.sequence.portal,base.model_ir_sequence,base.group_portal,1,0,0,0
+access_res_partner_bank_portal,access.res.partner.bank.portal,base.model_res_partner_bank,base.group_portal,1,0,0,0
+access_res_partner_portal,access.res.partner.portal,base.model_res_partner,base.group_portal,1,0,0,0
+access_ir_attachment_portal,access.ir.attachment.portal,base.model_ir_attachment,base.group_portal,1,0,0,0
+access_mail_message_portal,access.mail.message.portal,mail.model_mail_message,base.group_portal,1,0,0,0
```

Comment:
Is there a reason we need to add these permissions for models not in fieldservice?

Reply:
@brian10048 We are using the portal group to setup access from mobile app. I believe @murtuzasaleh got a permission error without this change.

Reply:
@brian10048 I got a permission error in the mobile app for the portal group.

Reply:
@brian10048 removed unused access rights.